bgpd/bgp_routemap.c

```diff
@@ -4309,9 +4309,11 @@ DEFUN (set_community_delete,
        "Delete matching communities\n")
 {
 	int idx_comm_list = 2;
+	char *args;
 
+	args = argv_concat(argv, argc, idx_comm_list);
```

Comment:
This function allocates memory that needs to be freed with MTYPE_TMP after use.

Reply:
damn, missed that :+1: